generators/csharp/codegen/src/project/CsharpProject.ts

```diff
@@ -198,6 +199,12 @@ export class CsharpProject extends AbstractProject<AbstractCsharpGeneratorContex
             join(absolutePathToProjectDirectory, RelativeFilePath.of(`${this.name}.csproj`)),
             templateCsProjContents
         );
+
+        await writeFile(
+            join(absolutePathToProjectDirectory, RelativeFilePath.of(`${this.name}.Custom.props`)),
```

Comment:
Should the file be named `Custom.Props` or `Custom.props`? The description below says `Custom.Props`, so as long as we're consistent.